### PR TITLE
Animated list items

### PR DESCRIPTION
**Pull request by pavelLyolya**

### CODE REVIEW

File: src/HOCs/AnimatedListItem.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    this.state = {
+      scaleValue: new Animated.Value(0),
+    }
+  }
+
+  componentDidMount() {
+    Animated.timing(this.state.scaleValue, {
+      toValue: 1,
+      duration : 600,
```
Comment (nataly-chobat) [inline src/HOCs/AnimatedListItem.js]: Additional space

File: src/HOCs/AnimatedListItem.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      delay: this.props.index * 40,
+      useNativeDriver: true,
```
Comment (nataly-chobat) [inline src/HOCs/AnimatedListItem.js]: Do you know what `useNativeDriver` means?

File: src/HOCs/AnimatedListItem.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  index: number,
+  children: any,
+};
+
+class AnimatedListItem extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      scaleValue: new Animated.Value(0),
```
Comment (nataly-chobat) [inline src/HOCs/AnimatedListItem.js]: In order to train skills in Animated API let's add vertical/horizontal appearance animation. I mean `translateX`, `translateY`. For example item will appear from top left corner

Comment (nataly-chobat) [reply src/HOCs/AnimatedListItem.js]: Or only on x direction

File: src/HOCs/AnimatedListItem.js (hunk 1 above)
Comment (pavelLyolya) [reply src/HOCs/AnimatedListItem.js]: what do you mean?

File: src/HOCs/AnimatedListItem.js (hunk 2 above)
Comment (pavelLyolya) [reply src/HOCs/AnimatedListItem.js]: of course!

**Pull request merged**